README.md

```diff
@@ -23,88 +28,133 @@ gem 'mongoid-locker'
 
-### Customizable :locked_at and :locked_until field names
-By default, Locker uses fields with `:locked_at` and `:locked_until` names which should be defined in a model.
+### Customizable :locking_name and :locked_at field names
+By default, Locker uses fields with `:locking_name` and `:locked_at` names which should be defined in a model.
 ```ruby
 class User
   include Mongoid::Document
   include Mongoid::Locker
 
+  field :locking_name, type: String
   field :locked_at, type: Time
-  field :locked_until, type: Time
 end
 ```
 
-Use `Mongoid::Locker.configure` to setup field names which used by Locker for all models where it's included.
+Use `Mongoid::Locker.configure` to setup parameters which used by Locker for all models where it's included.
 ```ruby
 Mongoid::Locker.configure do |config|
-  config.locked_at_field = :global_locked_at
-  config.locked_until_field = :global_locked_until
+  config.locking_name_field = :global_locking_name
+  config.locked_at_field    = :global_locked_at
 end
 
 class User
   include Mongoid::Document
   include Mongoid::Locker
 
+  field :global_locking_name, type: String
   field :global_locked_at, type: Time
-  field :global_locked_until, type: Time
 end
 ```
 
-The `locker` method in your model accepts `:locked_at_field` and `:locked_until_field` options to setup field names which used by Locker for the model. This can be useful when another library uses the same field for different purposes.
+The `locker` method in your model accepts options to setup parameters for the model.
 ```ruby
 class User
   include Mongoid::Document
   include Mongoid::Locker
 
+  field :locker_locking_name, type: String
   field :locker_locked_at, type: Time
-  field :locker_locked_until, type: Time
 
-  locker locked_at_field: :locker_locked_at,
-         locked_until_field: :locker_locked_until
+  locker locked_at_field: :locker_locking_name,
+         locked_at_field: :locker_locked_at
+end
+```
+
+### Available parameters for Mongoid::Locker, a class where it's included
+| parameter | default | options | description |
+|---|---|---|
+| locking_name_field | `:locking_name` | any field name | field where name of locking is storing, must be type of `String` |
+| locked_at_field | `:locked_at` | any field name | field where it is storing the time of beginning a lock of a document, must be type of `Time` |
+| lock_timeout | `5` | | within this time (in seconds) a document is considering as locked |
+| locker_write_concern | `{ w: 1 }` | see [MongoDB Write Concern](https://docs.mongodb.com/manual/reference/write-concern/#write-concern-specification)| a write concern only used for lock and unlock operations |
+| locking_name_length | `nil` | accepts `n` option for [SecureRandom.urlsafe_base64](https://ruby-doc.org/stdlib-2.2.3/libdoc/securerandom/rdoc/SecureRandom.html#method-c-urlsafe_base64) | uses by `secure_locking_name` generator |
```

Comment:
used by

Reply:
Maybe just spell it out? _the length, in bytes, of the random number to be generated_